docs/extensions/sqlite_scanner.md

```diff
@@ -118,5 +118,13 @@ SET GLOBAL sqlite_all_varchar=true;
 
 When set, this option overrides the type conversion rules described above, and instead always converts the SQLite columns into a `VARCHAR` column. Note that this setting must be set *before* `sqlite_attach` is called.
 
+## Data Types

```

Comment:
Can you review this header? Doesn't make a lot of sense given the contents of the section

Reply:
Yeh, good point. How about now?

Reply:
Looks good, thanks!